pkg/controller/cincinnati/new.go

```diff
@@ -301,6 +344,27 @@ func (k *kubeResources) newDeployment(instance *cv1alpha1.Cincinnati) *appsv1.De
 			},
 		},
 	}
+	if k.UseInitContainer(instance) {
+		dep.Spec.Template.Spec.InitContainers = []corev1.Container{
+			*k.graphDataInitContainer,
+		}
+	}
+
+	return dep
+}
+
+func (k *kubeResources) newGraphDataInitContainer(instance *cv1alpha1.Cincinnati, image string) *corev1.Container {
```

Comment:
```suggestion
func (k *kubeResources) newGraphDataInitContainer(instance *cv1alpha1.Cincinnati) *corev1.Container {
```

You already have the instance.